Attribute Analysis.ipynb

```diff
@@ -0,0 +1,650 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Attribute Analysis\n",
+    "The following notebook can analyze a given attribute with any number of expected values (2+), and will perform metrics 1,2,3,4,11 generalized to any attribute"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Instruction\n",
+    "1. Fill in the dataset in section 1.1\n",
+    "\n",
+    "2. Fill in attribute information in 1.2\n",
+    "\n",
+    "3. Run all of the cells\n",
+    "\n",
+    "4. Look at the summary pdf generated AND/OR explore each metric below.\n",
+    "    - Under each Metric there will be a portion of \"Setup\" and then \"Analyses\". Ignore the \"Setup\" unless customization is needed, and in \"Analyses\" results are shown to be interacted with. The number that comes after the M in the title refers to the measurement number when collecting the metrics.\n",
+    "    \n",
+    "## Table of Contents\n",
+    "1. [Initial Setup](#setup) <br/>\n",
+    "    1.1 [Dataset](#dataset) <br/>\n",
+    "    1.2 [Attribute](#attribute)\n",
+    "2. (M1) Metric: [Distance from center, size, attribute label inference](#metric1)<br/>\n",
+    "    2.1 [Setup](#metric1_setup)<br/>\n",
+    "    2.2 [Analyses](#metric1_analyses)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Initial Setup\n",
```

Comment:
for uniformity with other notebooks, I think this cell of "initial setup" is one font bigger, so just one "#"